### PR TITLE
IGNITE-7203 Java 8 by default

### PR DESCRIPTION
**Pull request by vveider**

### CODE REVIEW

File: modules/benchmarks/pom.xml [hunk 1]
```diff
@@ -35,7 +35,6 @@
     <url>http://ignite.apache.org</url>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jmh.version>1.13</jmh.version>
         <javac.target>1.7</javac.target>
```
Comment (ybabak) [inline modules/benchmarks/pom.xml]: is this correct?

File: examples/README-LGPL.txt [hunk 2]
```diff
@@ -11,9 +11,8 @@ The examples folder contains he following subfolders:
 - `config` - contains Ignite configuration files needed for examples.
 - `memcached` - contains PHP script demonstrating how Ignite Cache can be accessed using Memcached client.
 - `rest` - contains PHP script demonstrating how Ignite Cache can be accessed via HTTP API.
-- `src/main/java` - contains Java examples for different Ignite modules and features.
-- `src/main/java8` - contains additional set of Java examples utilizing Java 8 lambdas. These examples
-  are excluded by default, enable `java8` Maven profile to include them (JDK8 is required).
+- `src/main/java` - contains Java examples for different Ignite modules and features including utilizing
+   Java 8 lambdas ones.
```
[the remark this answers is not in the document]
Comment (vveider) [reply examples/README-LGPL.txt]: ML is now part of common examples, no need in separate mention

File: examples/README.txt [hunk 3]
```diff
@@ -11,9 +11,8 @@ The examples folder contains he following subfolders:
 - `config` - contains Ignite configuration files needed for examples.
 - `memcached` - contains PHP script demonstrating how Ignite Cache can be accessed using Memcached client.
 - `rest` - contains PHP script demonstrating how Ignite Cache can be accessed via HTTP API.
-- `src/main/java` - contains Java examples for different Ignite modules and features.
-- `src/main/java8` - contains additional set of Java examples utilizing Java 8 lambdas. These examples
-  are excluded by default, enable `java8` Maven profile to include them (JDK8 is required).
+- `src/main/java` - contains Java examples for different Ignite modules and features including utilizing
+   Java 8 lambdas ones.
```
[the remark this answers is not in the document]
Comment (vveider) [reply examples/README.txt]: ML is now part of common examples, no need in separate mention

File: examples/README.txt [hunk 4]
```diff
@@ -11,9 +11,8 @@ The examples folder contains he following subfolders:
...
 - `src/main/scala` - contains examples demonstrating usage of API provided by Scalar.
 - `src/main/java-lgpl` - contains lgpl-based examples for different Ignite modules and features.
 - `src/main/ml` - contains Java examples for Ignite-ML module.
```
[the remark this answers is not in the document]
Comment (vveider) [reply examples/README.txt]: Removed

File: examples/README.txt [hunk 5]
```diff
@@ -52,6 +45,6 @@ ML
 ==
 Math/Machine Learning examples can be activated by turning ml profile on.
```
[the remark this answers is not in the document]
Comment (vveider) [reply examples/README.txt]: Removed

File: modules/benchmarks/pom.xml (hunk 1 above)
Comment (vveider) [reply modules/benchmarks/pom.xml]: This is leftover properties section -- `javac.target` is no longer used, so it is removed, and jmh.version moved to parent.

**Comment by dspavlov** [comment]
The ticket is resolved. Closing PR

**Pull request closed without merging**